internal/adapters/handlers/jsonify.helper.go

```
@@ -0,0 +1,24 @@
+		return err
+	}
+
+	js = append(js, '\n')
+
+	for key, value := range headers {
+		w.Header()[key] = value
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(status)
+	w.Write(js)
```

golangci-lint

Error return value of `w.Write` is not checked

(errcheck)